portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +29,43 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+function addRandomFact() {
+    const facts = 
+        ['I am allergic to the allium family', 'I have backpacked over 400 miles', 
+        'My favorite sport is soccer'];
+
+    const fact = facts[Math.floor(Math.random() * facts.length)];
+
+    const factContainer = document.getElementById('fact-container');
+    factContainer.innerText = fact;
+}
+
+function setRandomStatus() {
```

Comment:
Rename function: the status is no longer random, it depends on which container the mouse is hovering over.